data_tools/add.py

```diff
@@ -83,5 +83,7 @@
         messages_to_write.append(message)
         added += 1
     
+    print ("To add: {}, Skipping: {}".format(added, skipped_existing))
```

Comment:
```suggestion
    print(f"To add: {added}, Skipping: {skipped_existing}")
```

Reply:
thanks @as2388 - I was following the print format style in the rest of the file even though I generally prefer your suggested format as well. Perhaps it might be better to update those as well? plmk if you'd like me to do it in this PR or separately

Reply:
Ah I see, that makes sense. I think better in a later (low priority) PR?

Reply:
yup, sounds good, thanks. Issue opened for tracking: https://github.com/AfricasVoices/CodaV2/issues/266